SPECS/tidy/tidy.spec

```
@@ -92,6 +93,9 @@ rm -fv %{buildroot}%{_libdir}/libtidy.a
 %{_libdir}/pkgconfig/tidy.pc
 
 %changelog
+* Mon May 01 2023 Sean Dougherty <sdougherty@microsoft.com> - 5.8.0-6
+- Added bespoke backport patch to fix CVE-2021-33391
```

```suggestion
- Backported patch to fix CVE-2021-33391
```
nit - suggestion for more concise wording